lib/operations/accessor/missing.js

```diff
@@ -0,0 +1,33 @@
+import isArray from '../../helpers/isArray';
+
+function missing(apply) {
+  /*
+  Missing can receive many keys as many arguments, like {"missing:[1,2]}
+  Missing can also receive *one* argument that is an array of keys,
+  which typically happens if it's actually acting on the output of another command
+  (like 'if' or 'merge')
+  */
+  var are_missing = [];
+
+  for (var _len = arguments.length, args = new Array(_len > 1 ? _len - 1 : 0), _key = 1; _key < _len; _key++) {
+    args[_key - 1] = arguments[_key];
+  }
+
+  var keys = isArray(args[0]) ? args[0] : args;
+
+  for (var i = 0; i < keys.length; i++) {
+    var key = keys[i];
+    var value = apply({
+      var: key
+    }, this);
+
+    if (value === null || value === '') {
+      are_missing.push(key);
+    }
+  }
+
+  return are_missing;
+}
+
+missing.withApply = true;
```

Comment:
@lina-salih 
@mjenny 
I don't like this hacks of meta data like `withApply` and `code`...
The reason I have them is, that some operations need to call `apply` recursively, and others have names within the API like `==` which is not a valid JavaScript function or variable name., so:
- `withApply`: adds the `apply` function as a callback as the first argument
- `code`: overrides the functions key within the `operations` object, e.g. `equal()` function gets overridden by it's `code` property being `==`

Reply:
Do you plan on changing this at some point?

Reply:
Yeah, it sit's on my nerves^^

Reply:
Go for it. 💯

Reply:
Oh.... do you want to change this in the current PR or open an issue? 😖